apps/finance/test/finance.js

```diff
@@ -1,990 +1,1608 @@
-const getBalance = require('@aragon/test-helpers/balance')(web3)
```

Comment:
This test file change is very difficult to review.

Please—it's easy to make multiple commits with git and we should take advantage of this to make reviewing these types of changes much easier than re-reading 2500 lines of code.

Can we revert the changes here, and then apply in separate commits:

1. Update formatting
2. Test changes

Reply:
@sohkai Apologies for that. It's ready to review now. I only included the changes to move to web3@1 now. I had a problem with one extra test than Ale. 

`sets the end of time correctly`

I'm including it to #1111